lib/r10k/cli/deploy.rb

```diff
@@ -31,6 +31,13 @@ def self.command
             exit 1
           end
         end
+        option nil, :'puppet-conf', 'Path to puppet.conf', argument: :required do |value, cmd|
+          unless File.readable? value
+            $stderr.puts "The specified puppet.conf #{value} could not be read."
```

Comment:
We can use the `logger` method as it's defined on commands; the following should work:

```suggestion
            logger.fatal "The specified puppet.conf #{value} could not be read."
```

(We should also apply the same change for `puppet-path`, but that's best tracked in another PR.)

Reply:
Made this change.